src/energyplus/Test/AirflowNetwork_GTest.cpp

```diff
@@ -200,3 +225,250 @@ TEST_F(EnergyPlusFixture, ForwardTranslator_AirflowNetworkEquivalentDuct) {
 
   // workspace.save(toPath("./AirflowNetworkLinkage.idf"), true);
 }
+
+TEST_F(EnergyPlusFixture, ForwardTranslator_AirflowNetworkSurfaces) {
+  // SimpleOpening
```

Comment:
Testing all these classes in the same test isn't ideal, as it makes it less clear which class is at fault if it starts failing. We'll be able to track the line number and see, so I'm fine letting this slide though.